modin/engines/base/io/text/text_file_dispatcher.py

```diff
@@ -258,3 +259,157 @@ def _read_rows(
             rows_read += 1
 
         return outside_quotes, rows_read
+
+    # _read helper functions
+    @classmethod
+    def rows_skipper_builder(cls, f, quotechar, is_quoting):
+        def skipper(n):
+            if n == 0 or n is None:
+                return 0
+            else:
+                return cls._read_rows(
+                    f,
+                    quotechar=quotechar,
+                    is_quoting=is_quoting,
+                    nrows=n,
+                )[1]
+
+        return skipper
+
+    @classmethod
+    def _define_header_size(
+        cls,
+        header: Union[int, Sequence[int], str, None] = "infer",
```

Comment:
I dont believe header can be `None`? Or is it [undocumented behavior](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_csv.html)?

Reply:
Documentation mentions fractionally `header=None` case in the string ` if column names are passed explicitly then the behavior is identical to header=None`.
`heder=None` can be set, for example, to ignore inferring of header row:
```
import pandas
from io import StringIO

data = "1,2,3\n4,5,6\n"

df = pandas.read_csv(StringIO(data), header=None)
print(df)

Output:
   0  1  2
0  1  2  3
1  4  5  6
```

Reply:
Ah ok my mistake! This looks good then!